layouts/_default/single.html

```diff
@@ -3,28 +3,34 @@
 {{ end }}
 
 {{ define "main" }}
-{{ $dateFormat := .Site.Params.dateFormat | default "Jan 2 2006" }}
+    {{ $dateFormat := .Site.Params.dateFormat | default "Jan 2 2006" }}
```

Comment:
Formatting has been an issue in this project I've noticed. Probably a lack of symmetry in formatting tools. Maybe we can consider something like `prettier` in the future.